test-data/unit/check-expressions.test

```diff
@@ -1157,6 +1157,15 @@ b'%c' % (123)
 [case testUnicodeInterpolation_python2]
 u'%s' % (u'abc',)
 
+
+-- F-String
+-- --------
+
+
+[case testFStringParseOk]
+a = f'foobar'
```

Comment:
You need `# flags: --fast-parser` here (see how other tests use it) else you get a SyntaxError.

Can you also add tests showing that the expressions in `{}` are actually type-checked?